CHANGELOG.md

```diff
@@ -11,16 +11,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Added
 
+- `--cores` flag to specify the number of cores to use for parallel execution
+- `#[ignore]` attribute together with `--ignored` and `include-ignored` flags - read more [here](https://foundry-rs.github.io/starknet-foundry/testing/testing.html#ignoring-some-tests-unless-specifically-requested)
```

Comment:
Not sure about including links in this changelog. The will eventually stop working if/once something in the docs is changed

Reply:
It is just for the next release, I think we did that before with testing internals. If it stops working on release 0.10 no one will care

Reply:
I agree with @piotmag769, including links is a very good idea!